OpenEdXMobile/src/main/java/org/edx/mobile/whatsnew/WhatsNewModel.java

```diff
@@ -4,68 +4,49 @@
 import android.os.Parcelable;
 import android.support.annotation.NonNull;
 
+import java.util.List;
+
 public class WhatsNewModel implements Parcelable {
     @NonNull
-    private String title;
-    @NonNull
-    private String message;
-    @NonNull
-    private String image;
-
+    private String version;
     @NonNull
-    public String getTitle() {
-        return title;
-    }
+    private List<WhatsNewItemModel> messages;
```

Comment:
`messages` can be renamed to `whatsNewModels` or to some more relevant name.

Reply:
Its dependent on json key i-e- `messages`.
Actually we are using Androids Parceling for automatic json to object mapping which needs exact same key and variable name.